api.ts

```diff
@@ -2439,26 +2430,7 @@ export function makeSyncTable<
       }
     : undefined;
 
-  const executeGetPermissions = wrappedExecuteGetPermissions
-    ? async function execGetPermissions(
-        params: ParamValues<ParamDefsT>,
-        request: ExecuteGetPermissionsRequest<K, L, SchemaDefT>,
-        context: GetPermissionExecutionContext,
-      ) {
-        const result = await wrappedExecuteGetPermissions(params, request, context);
-        const {rowAccessDefinitions: permissions} = result;
-        const oversizedRowAccessDefinitions = permissions.filter(p => p.permissions.length > MaxPermissionsPerRow);
-
-        if (oversizedRowAccessDefinitions.length > 0) {
-          throw new Error(
-            `Objects with ids: ${oversizedRowAccessDefinitions
-              .map(p => p.rowId)
-              .join(', ')} returned more permissions than the maximum allowed of ${MaxPermissionsPerRow} per object`,
-          );
-        }
-        return result;
-      }
-    : undefined;
+  const executeGetPermissions = wrappedExecuteGetPermissions ? wrappedExecuteGetPermissions : undefined;
```

Comment:
Do you even need this? Just rename `wrappedExecuteGetPermissions` to `executeGetPermissions` and return it.

Reply:
We do not